files/scripts/syncd.sh

```diff
@@ -11,7 +11,7 @@ function collect_saisdkdump() {
     /usr/bin/docker exec syncd$DEV rm -rf ${TMP_DMP_DIR}
     /usr/bin/docker exec syncd$DEV rm -f /tmp/${sai_dump_filename_epoch}.tar
     /usr/bin/docker exec syncd$DEV mkdir -p ${TMP_DMP_DIR}
-    /usr/bin/docker exec syncd$DEV saisdkdump -f ${TMP_DMP_DIR}/${sai_dump_filename} > /dev/null
+    timeout 1m bash -c "/usr/bin/docker exec syncd$DEV saisdkdump -f ${TMP_DMP_DIR}/${sai_dump_filename} > /dev/null"
```

Comment:
1 minute seems to be a huge timeout. Can you check how long saisdkdump takes in each platform and come up with 2x of max time taken. Ideally saisdkdump should finish within 5-6 seconds. If that's the scenario we can add timeout of 15 seconds

Reply:
This is worst case scenario right. And we can't predict how long will the saisdkdump of other vendors would take. Anyways, i'll set it to 30s. Seems reasonable?